sources/packages/backend/apps/db-migrations/src/migrations/1705441135641-RemovePDVerifiedColumn.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class RemovePDVerifiedColumn1705441135641 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData("Remove-StudentPDVerified.sql", "Student"),
```

Comment:
We should follow the kebab case for "Remove-StudentPDVerified.sql" and "Add-StudentPDVerified.sql".

Reply:
yes, that file already existed than I didn't want to change its name but I think I can do that with no harm if I change the references. Will change that.

Reply:
I reverted the changes for that file and created a new one with kebab case.